tracer.go

```diff
@@ -145,6 +145,11 @@ type StartSpanOptions struct {
 	// If specified, the caller hands off ownership of Tags at
 	// StartSpan() invocation time.
 	Tags map[string]interface{}
+
+	// Perfevent is the platform metric's name(s) or an event name
+	// supported by perfevents package
+	// (https://github.com/opentracing-contrib/perfevents/go)
+	Perfevent string
```

Comment:
* nit: PerfEvent
 * it's not clear what tracer is expected to do when receiving this option in the StartOptions.
 * instead of extending the StartOption API, can this instead be passed as one of the Tags? We already have precedent for that, `sampling.priority` tag.